hathor/nanocontracts/blueprint.py

```diff
@@ -91,6 +111,27 @@ def __new__(cls, name, bases, attrs, **kwargs):
                 # This was not implemented yet and will be extended later.
                 raise BlueprintSyntaxError(f'fields with default values are currently not supported: `{field_name}`')
 
+        # validation makes sure we already have it
+        original_init_fn = getattr(new_class, NC_INITIALIZE_METHOD)
+        init_containers_fn = _make_initialize_uninitialized_container_fields_fn(container_fields)
+
+        # patch initialize method so it initializes containers fields implicitly
+        @wraps(original_init_fn)
+        def patched_init_fn(self: Blueprint, *args: Any, **kwargs: Any) -> Any:
+            ret = original_init_fn(self, *args, **kwargs)
+            init_containers_fn(self)
+            return ret
```

Comment:
Shouldn't we initialize the containers before calling the `initialize()` method?